### DISCUSSION TITLE
Telegram/Discord server for chatting?

### DISCUSSION THREAD

**Discussion by herold33** [kyujin-cho/pixel-volte-patch#69] [General]
Maybe let's build a community around this? Any discord/telegram channel would be a great starting point 😇

**Comment by andresro14** [comment]
I agree with this. It would be nice also to know if it's still working after monthly updates.

**Comment by kyujin-cho** [comment]
Hi! ATM there's no plan to run another community about the application other than this discussions tab. If you need a faster response please do not hesitate to mention me (@kyujin-cho), as this is the only way for me to receive a notification from mobile Github app.
